tests/src/test/scala/cats/mtl/tests/Syntax.scala

```diff
@@ -59,6 +68,12 @@ final class Syntax extends BaseSuite {
       def fb[F[_]: Raise[?[_], EE], E <: EE, EE](e: E): F[E] = e.raise[F, E]
       val faC: Either[String, Int] = Raise.raise[EitherC[String]#l, String, Int]("ha")
       val faeC: Either[String, Nothing] = Raise.raiseF[EitherC[String]#l]("ha")
+
+      def bar[F[_]](implicit F: Raise[F, Bar]) =
+        F.raise(Bar(404))
+
+      def foo[F[_]](implicit F: Raise[F, Foo]): F[Nothing] =
+        bar[F]
```

Comment:
Another case that would be nice to add is to force the compiler to do subtyping along both axes:

```scala
def foo[F[_]](implicit F: Handle[F, Throwable]) = 
  new RuntimeException("yo").raise[F]
```

Reply:
Actually I discovered this case already exists in the syntax tests, so we're good.